server/src/server.ts

```diff
@@ -0,0 +1,55 @@
+import { createServer, IncomingMessage, ServerResponse } from "http";
+import fs from "fs";
+import { parse, ParsedQuery } from "query-string";
+
+const port = 4000;
+
+const isPerson = (obj: any): boolean => {
+  return obj.name.length > 0 && Date.parse(obj.date) > 0;
+};
+
+createServer((req: IncomingMessage, res: ServerResponse): void => {
+  res.setHeader("content-type", "text/plain");
+  res.setHeader("Access-Control-Allow-Origin", "*");
+  res.setHeader(
+    "Access-Control-Allow-Headers",
+    "Content-Type, Access-Control-Allow-Headers, Authorization, X-Requested-With"
+  );
+  switch (req.method) {

```

Comment:
Switch case is a poor choice. We need allow only POST request. Other ones should be rejected with 403 error code.

Reply:
Changed